app/src/main/res/xml/preferences.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
```

Comment:
Is this need 🤔  Probably this is not layout file 😄

Reply:
Good catch. Not needed.